torch/jit/quantized.py

```diff
@@ -204,6 +206,234 @@ def forward(self, input, hx=None):
             self.zero_point_hh
         )
 
+@torch.jit.script
+def apply_permutation(tensor, permutation, dim=1):
+    # type: (Tensor, Tensor, int) -> Tensor
+    return tensor.index_select(dim, permutation)
+
+class QuantizedRNNBase(torch.jit.ScriptModule):
```

Comment:
Why did you have to copy all of `nn.LSTM` instead of inheriting from it and adding the `_pack` and `_unpack` methods?

Reply:
It's not just `_pack` and `_unpack`, this has a totally new constructor and different attributes. Basically the only things that could be inherited are some of the auxiliary helper methods, which I didn't think were valuable enough